doc/release-process.md

```diff
@@ -160,6 +160,9 @@ Codesigner only: Sign the windows binaries:
     Enter the passphrase for the key when prompted
     signature-win.tar.gz will be created
 
+Codeisnger only: It is advised to test that the code signature attaches properly prior to tagging by performing the `guix-codesign` step.
```

Comment:
s/Codeisnger/Codesigner/

Reply:
Each time I read "codesigned" I first see "co-designed" and have to push that interpretation away :D 

Writing "code signer" for the noun and "code-signed" / "code-signing" etc for the verb would be clearer to me.

Reply:
Fixed